setup.py

```diff
@@ -0,0 +1,113 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+
+from setuptools import find_packages
+
+# try:
+#     from skbuild import setup
+# except ImportError:
+#     sys.stderr.write("""scikit-build is required to build from source or run tox.
+# Please run:
+#   python -m pip install scikit-build
+# """)
+#     # from setuptools import setup
+#     sys.exit(1)
+from setuptools import setup
+
+
+with open('README.rst', 'rt') as readme_file:
+    readme = readme_file.read()
+
+
+def prerelease_local_scheme(version):
+    """
+    Return local scheme version unless building on master in CircleCI.
+
+    This function returns the local scheme version number
+    (e.g. 0.0.0.dev<N>+g<HASH>) unless building on CircleCI for a
+    pre-release in which case it ignores the hash and produces a
+    PEP440 compliant pre-release version number (e.g. 0.0.0.dev<N>).
+    """
+    from setuptools_scm.version import get_local_node_and_date
+
+    if os.getenv('CIRCLE_BRANCH') in {'master'}:
+        return ''
+    else:
+        return get_local_node_and_date(version)
+
+
+setup(
+    name='podo',
+    use_scm_version={'local_scheme': prerelease_local_scheme},
+    description='A Python toolkit for Histopathology Image Analysis',
+    long_description=readme,
+    long_description_content_type='text/x-rst',
+    author='Kitware, Inc.',
+    author_email='developers@digitalslidearchive.net',
+    url='https://github.com/DigitalSlideArchive/HistomicsTK',
+    packages=find_packages(exclude=['tests', '*_test']),
+    package_dir={
+        'podo': 'podo',
+    },
+    include_package_data=True,
+    install_requires=[
+        # scientific packages
+        'nimfa>=1.3.2',
+        'numpy==1.19.5',
+        'scipy>=0.19.0',
+        'Pillow==9.5.0',
+        'pandas==1.1.5',
+        'imageio>=2.3.0',
+        'shapely[vectorized]',
+        #'opencv-python-headless<4.7',
+        #'sqlalchemy',
+        'matplotlib',
```

Comment:
We shouldnt need this, please let us know why we need to save roi's in the disc. Ideally we should avoid that.

Reply:
Updated the code to not save roi's, It was for testing locally. Removed matplotlib as well.